src/datasources/human-description-api/json/contract-descriptions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "function transfer(address, uint256)": "Send {{tokenValue $1}} to {{address $0}}",
+  "function approve(address, uint256)": "Approve {{tokenValue $1}} to {{address $0}}",
+  "function withdraw(uint256)": "Withdraw {{tokenValue $0}}",
+  "function changeThreshold(uint256)": "Change threshold to {{decimals $0}}",
+  "function addOwnerWithThreshold(address,uint256)": "Add new owner {{address $0}} with threshold {{decimals $1}}",
+  "function removeOwner(address,address,uint256)": "Remove owner {{address $1}} with threshold {{decimals $2}}",
+  "function swapOwner(address,address,address)": "Swap owner {{address $1}} with {{address $2}}",
+  "function setGuard(address)": "Set guard {{address $0}}",
+  "function enableModule(address)": "Enable module {{address $0}}",
+  "function setFallbackHandler(address)": "Set fallback handler {{address $0}}",
+  "function setDelegate(bytes32, address)": "Set delegate {{address $1}}",
+  "function multiSend(bytes)": "Interaction with MultiSend",
+  "function setAllowance(address,address,uint96,uint16,uint32)": "Set allowance for {{address $0}}"
```

Comment:
Do we not want to support ERC-721?

It's not clear which of these are ERC-20/Safe. I understand that it was requested to merge all methods into one JSON, but perhaps creating a pre-build script to do so is a cleaner approach. What do you think?

Reply:
I would also imagine if this was to be extracted into an external repository it would be nicer to have those messages split by protocol. Wdyt @katspaugh ?

Reply:
Perhaps a JS module like this:

```js
export default {
    // ERC20
   "function transfer(address, uint256)": "Send {{tokenValue $1}} to {{address $0}}",
   "function approve(address, uint256)": "Approve {{tokenValue $1}} to {{address $0}}",
   
   // Safe
   "function changeThreshold(uint256)": "Change threshold to {{decimals $0}}",
   ...
}
```

Reply:
Btw, why is threshold a "decimals"?

Reply:
Its the type we defined for any number that is not a tokenValue. We could also call it `number`. Wdyt?

Reply:
In this case it would be specifically an integer since it's not divided by any "decimals".

Reply:
I've changed it to `number` and also the type from `unknown` to `bigint` because so far we are always using it on `uint256` which viem decodes as a BigInt. We can revisit this later if more types need to be added.